<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>User_MainWindow</class>
 <widget class="QMainWindow" name="User_MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>650</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>单词消除小游戏</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="main_tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>610</width>
      <height>410</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>610</width>
      <height>410</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>610</width>
      <height>410</height>
     </size>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <widget class="QWidget" name="game_tab">
     <attribute name="title">
      <string>游戏</string>
     </attribute>
     <widget class="QWidget" name="widget_3" native="true">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>30</y>
        <width>311</width>
        <height>311</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="new_game_pushButton">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>新游戏</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="continue_game_pushButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>继续游戏</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="multi_player_pushButton">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>多人对战</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="add_words_pushButton">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>开始出题</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="widget_5" native="true">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>110</y>
        <width>82</width>
        <height>77</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>关卡号：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="level_spinBox">
         <property name="maximum">
          <number>15</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="search_tab">
     <attribute name="title">
      <string>查找</string>
     </attribute>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>0</y>
        <width>251</width>
        <height>371</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="3" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0">
        <widget class="QWidget" name="widget_4" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="search_pushButton">
            <property name="text">
             <string>查找</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QWidget" name="widget" native="true">
         <layout class="QGridLayout" name="gridLayout">
          <item row="4" column="1">
           <widget class="QLineEdit" name="level_lineEdit"/>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>角色：</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLineEdit" name="passedlevels_lineEdit"/>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="usr_lineEdit"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>用户名：</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QWidget" name="widget_2" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QComboBox" name="role_comboBox">
               <item>
                <property name="text">
                 <string>闯关者</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>出题者</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>已闯关卡数：</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>出题数目：</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QLineEdit" name="questioncnt_lineEdit"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>等级：</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QTableView" name="search_tableView">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>10</y>
        <width>371</width>
        <height>361</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="challenger_ranking_tab">
     <attribute name="title">
      <string>闯关者排行榜</string>
     </attribute>
     <widget class="QTabWidget" name="challenger_tabWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>600</width>
        <height>370</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>370</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>600</width>
        <height>370</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="passed_levels_tab">
       <attribute name="title">
        <string>已闯关卡数</string>
       </attribute>
       <widget class="QTableView" name="challenger_tableView_1">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>600</width>
          <height>330</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>600</width>
          <height>330</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>600</width>
          <height>330</height>
         </size>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="challenger_level_tab">
       <attribute name="title">
        <string>等级</string>
       </attribute>
       <widget class="QTableView" name="challenger_tableView_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>600</width>
          <height>330</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>600</width>
          <height>330</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>600</width>
          <height>330</height>
         </size>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QPushButton" name="crank_ref_pushButton">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>0</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>刷新</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="questioner_ranking_tab">
     <attribute name="title">
      <string>出题者排行榜</string>
     </attribute>
     <widget class="QTabWidget" name="questioner_tabWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>600</width>
        <height>370</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>370</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>600</width>
        <height>370</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="question_cnt_tab">
       <attribute name="title">
        <string>出题数目</string>
       </attribute>
       <widget class="QTableView" name="questioner_tableView_1">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>600</width>
          <height>330</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>600</width>
          <height>330</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>600</width>
          <height>330</height>
         </size>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="questioner_level_tab">
       <attribute name="title">
        <string>等级</string>
       </attribute>
       <widget class="QTableView" name="questioner_tableView_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>600</width>
          <height>330</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>600</width>
          <height>330</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>600</width>
          <height>330</height>
         </size>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QPushButton" name="qrank_ref_pushButton">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>0</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>刷新</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>650</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
